docs/use_cases.rst

```diff
@@ -1,15 +1,16 @@
 NIX in action
 =============
 
-We will illustrate the use of NIX in three different contexts:
+We will illustrate the use of NIX in different contexts:
 
 1) Electrophysiological recordings and intergration of NIX into a recording tool
 2) Use of NIX as a storage backend for the `neo <http://neuralensemble.org/neo>`__ data model.
 3) Use of NIX to store EEG data.
-
+4) Use of NIX as a storage backend for `SpikeInterface <https://spikeinterface.readthedocs.io/en/latest/>`__.
 
 .. toctree::
    relacs
    neonixio
    eegbase
+   spikeinterface
```

Comment:
I guess I'll have to write this (and the Neo one) at some point? :)

Reply:
yes, that would be most welcome ;)